packages/react-scripts/config/webpack.config.ssr.js

```diff
@@ -655,9 +677,12 @@ module.exports = function(webpackEnv) {
       // It is absolutely essential that NODE_ENV is set to production
       // during a production build.
       // Otherwise React will be compiled in the very slow development mode.
-      new webpack.DefinePlugin(env.stringified),
+      new webpack.DefinePlugin({
+        ...env.stringified,
+        'typeof window': '"undefined"',
+      }),
       // This is necessary to emit hot updates (currently CSS only):
-      isEnvDevelopment && new webpack.HotModuleReplacementPlugin(),
+      // isEnvDevelopment && new webpack.HotModuleReplacementPlugin(),
```

Comment:
Could you confirm why this has been removed?

Reply:
As above re: style, and I don't _think_ https://webpack.js.org/concepts/hot-module-replacement/ is designed for the server.